app/models/openscap_result.rb

```diff
@@ -0,0 +1,68 @@
+require 'openscap'
+require 'openscap/ds/arf'
+require 'openscap/xccdf/benchmark'
```

Comment:
Sorry, @moolitayer, it's an observation that I didn't explain well.  What I mean is we have 3 requires for 66 lines of code.  If need to add more code here, will we need to add more requires?  If these requires are a common pattern, a loader in openscap (such as the require 'openscap' itself) could require them for us.  It doesn't have to happen, it's only an observation.